src/chat/_chatwidget.py

```diff
@@ -238,6 +273,14 @@ def on_identified(self):
         #Perform any pending autojoins (client may have emitted autoJoin signals before we talked to the IRC server)
         self.autoJoin(self.optionalChannels)
         self.autoJoin(self.crucialChannels)
+        self.autoJoin(self.autoJoinChannels)
+        
+        #this might be timing dependant
+        if self.client.useLanguageChannel and self.client.me.country and self.client.me.country in self.languageChannels:
+            self.autoJoin(self.languageChannels[self.client.me.country])
+
+        if self.client.useNewbiesChannel and 100 < self.client.me.global_rating[1] < 500:
```

Comment:
This currently moves people in the newbie channel if their deviation is > 100, i'm not quite sure how good this value is